tools/default_settings.py

```diff
@@ -13,55 +16,30 @@
 """
 
 from os.path import join, abspath, dirname
-import logging
 
-ROOT = abspath(join(dirname(__file__), ".."))
-
-# These default settings have two purposes:
-#    1) Give a template for writing local "private_settings.py"
-#    2) Give default initialization fields for the "toolchains.py" constructors
+ROOT = abspath(join(dirname(__file__), "."))
 
 ##############################################################################
 # Build System Settings
 ##############################################################################
-BUILD_DIR = abspath(join(ROOT, "build"))
+#BUILD_DIR = abspath(join(ROOT, "build"))
 
 # ARM
-ARM_PATH = "C:/Program Files/ARM"
-ARM_BIN = join(ARM_PATH, "bin")
-ARM_INC = join(ARM_PATH, "include")
-ARM_LIB = join(ARM_PATH, "lib")
-
-ARM_CPPLIB = join(ARM_LIB, "cpplib")
-MY_ARM_CLIB = join(ARM_PATH, "lib", "microlib")
+#ARM_PATH = "C:/Program Files/ARM"
```

Comment:
one day i'll convince you to set this path for Keil MDK rather than just the compiler... one day...